packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -77,6 +84,23 @@ const MarketPlacePage = () => {
     'error'
   );
 
+  const handleInputChange = (input) => {
+    setSearchQuery(input);
+    setSearchResults(searchIndex.search(input));
+  };
+
+  useEffect(() => {
+    if (isLoading) return;
+    marketplacePluginsResponse.data.forEach((plugin) => {
+      const fieldsToIndex = JSON.stringify({
```

Comment:
Same comment as the other PR, we should not use JSON stringify here to avoid indexing the "name" and "description" keys